Source/Plugins/Core/com.equella.core/js/__tests__/tsrc/search/SearchPage.test.tsx

```diff
@@ -308,13 +301,52 @@ describe("Hide Refine Search controls", () => {
     jest.clearAllMocks();
   });
 
-  const getOwnerSelector = () =>
-    queryRefineSearchComponent(page.container, "OwnerSelector");
-  const getDateSelector = () =>
-    queryRefineSearchComponent(page.container, "DateRangeSelector");
-  const getStatusSelector = () =>
-    queryRefineSearchComponent(page.container, "StatusSelector");
+  it("Should contain the correct controls", async () => {
+    const pageContainer = page.container;
+    const findCollapsibleSection = page.container.getElementsByClassName(
```

Comment:
Nit: This name makes it sound like a function. Maybe:

```suggestion
    const collapsibleSections = page.container.getElementsByClassName(
```

Is there a suitable RTL query we could do here directly on `page` rather than using container. Probably a `getAllBy`?

Reply:
Currently, the only way to target the collapsible filter section is by using a classname. There's no RTL query to perform this function. So it will have to stay as-is

Reply:
I would try to give an ID to the panel, get the panel by `querySelector`, and then use `getAllByRole("listitem")` to get a list of controls wrapped inside this panel.

Reply:
You may not even need an ID. You can get the panel by finding the parent node of  h5 `Refine search`.